gammapy/maps/wcsnd.py

```diff
@@ -508,3 +508,27 @@ def plot(self, ax=None, idx=None, **kwargs):
         im = ax.imshow(data, **kwargs)
         ax.coords.grid(color='w', linestyle=':', linewidth=0.5)
         return fig, ax, im
+
+
+    def make_region_mask(self, region, inside=True):
+        """Create a mask of a given region
+
+        TODO: implement list of region for each axis
+
+        Parameters
+        ----------
+        region :  `~regions.PixelRegion` or `~regions.SkyRegion` object
+            A region on the sky could be defined in pixel or sky coordinates.
+        inside : bool
+          Output map is True inside the input region if inside is set to True and False outside and conversely.
+
+        Return
+        ------
+        mask_map : `~gammapy.maps.WcsNDMap`
+            the mask map
+        """
+        mask = self.geom.get_region_mask_array(region)
```

Comment:
Can't you pass `inside` to `geom.get_region_mask_array` in order to prevent the duplicated check for ``if inside`` in the following lines?

Reply:
OK.